pkg/kubelet/api/v1alpha1/lifecycle/api.pb.go

```diff
@@ -117,20 +117,23 @@ func (CgroupInfo_Kind) EnumDescriptor() ([]byte, []int) { return fileDescriptorA
 type IsolationControl_Kind int32
 
 const (
-	IsolationControl_CGROUP_CPUSET_CPUS IsolationControl_Kind = 0
-	IsolationControl_CGROUP_CPUSET_MEMS IsolationControl_Kind = 1
-	IsolationControl_CONTAINER_ENV_VAR  IsolationControl_Kind = 2
+	IsolationControl_CGROUP_CPUSET_CPUS   IsolationControl_Kind = 0
+	IsolationControl_CGROUP_CPUSET_MEMS   IsolationControl_Kind = 1
+	IsolationControl_CONTAINER_ENV_VAR    IsolationControl_Kind = 2
+	IsolationControl_CGROUP_HUGETLB_LIMIT IsolationControl_Kind = 3
```

Comment:
For each `IsolationControl.Kind`, we should include docs about how to encode values for the controller in `value` and `map_value` fields (including examples). I should have done this when adding the `CONTAINER_ENV_VAR` kind in retrospect. This protobuf should be the final word on the Isolator API semantics IMO.